### ISSUE TITLE
Add auto-backup feature

### ISSUE THREAD

**Issue by acsor** [photodiode/panorama-view#98]
From the edit of a review I left on the addons store:

>Would you mind adding an auto-backup feature? Too many a time have I crashed my browser and lost all of my tabs. Having no backup at hand, and no way of restoring my previous windows session, I had to start over collecting and organizing my tabs.

That's it. Do you find possible to have such functionality? Can you require permissions to write to disk periodically? This feature would be a real time-saver, as crashing one's browser or abrubptly closing it occurs more frequently than you can imagine.

**Comment by qrospars** [comment]
Totally agree. I will try to code it and make a pull request since I also really want this feature.

**Comment by photodiode** [comment]
I don't have much time to work on it at the moment, so you're very welcome to give it a shot!
The file handling is more complicated than I had hoped..

**Comment by acsor** [comment]
Hi, I have noticed, maybe with a certain delay, that the feature was finally implemented into the add on. That's great! However, as of now, I only see the possibility to specify an interval no greater than 6h. Would it be possible for you to enlarge this interval so that it's possible to set it to one or more days as well? I'd personally like to back up my tabs once a day, but no more.

Thanks for your time and effort!
